tests/Output/UnifiedDiffOutputBuilderDataProvider.php

```diff
@@ -0,0 +1,400 @@
+<?php declare(strict_types=1);
+/*
+ * This file is part of sebastian/diff.
+ *
+ * (c) Sebastian Bergmann <sebastian@phpunit.de>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace SebastianBergmann\Diff\Output;
+
+/**
+ * @coversNothing
+ */
+final class UnifiedDiffOutputBuilderDataProvider
+{
+    public static function provideDiffWithLineNumbers(): array
```

Comment:
what is the reason to extract only this one provider to separated pseudo-class ?

Reply:
The data is used in two tests. I used to import/include one test in the other, but that causes PHPUnit to execute only one of the two tests.
(same issue as we had here https://github.com/FriendsOfPHP/PHP-CS-Fixer/commit/d3924aacdfc5ac2ec6373293a5babd5bafa6ed0c#diff-c3a69cf56955b7742f451cb62963a73dR332)